.vscode/settings.json

```diff
@@ -1,3 +1,12 @@
 {
-	"C_Cpp.intelliSenseEngine": "disabled",
-}
```

Comment:
This looks like you changed all the line endings.
Make sure git autocrlf is properly set up.

Reply:
fixed